tests/unit/test_sde.py

```diff
@@ -170,10 +174,13 @@ def test_KL_sde(setup):
     # convert OU-SDE to SSM
     A_p = -1 * ou_sde.decay * tf.ones_like(m)[..., None] * dt + tf.eye(1, dtype=DTYPE)
     b_p = tf.zeros_like(m) * dt
-    p_ssm = state_space_model_from_covariances(initial_mean=x0, initial_covariance=1e-1 * tf.ones_like(x0[..., None]),
-                                               state_transitions=A_p,
-                                               state_offsets=b_p,
-                                               process_covariances=tf.square(ou_sde.diffusion(m, t=None)) * dt)
+    p_ssm = state_space_model_from_covariances(
+        initial_mean=x0,
+        initial_covariance=1e-1 * tf.ones_like(x0[..., None]),
+        state_transitions=A_p,
+        state_offsets=b_p,
+        process_covariances=tf.square(ou_sde.diffusion(m, t=None)) * dt,
+    )
 
     # generate a q SSM and q SDE
```

Comment:
so p is deterministic and q is sampled from?

Reply:
In essence, the test function generates two SSMs from OU SDE and tests the KL value. Both the SSMs are generated by randomly initializing the decay value of the drift function.